src/bin/cosmogony.rs

```diff
@@ -38,6 +67,24 @@ Accepted extensions are '.json', '.json.gz', '.jsonl', '.jsonl.gz'
     filter_langs: Vec<String>,
 }
 
+#[derive(StructOpt, Debug)]
+struct MergeArgs {
+    /// Cosmogony files to process
+    #[structopt(name = "FILE", parse(from_os_str))]
+    files: Vec<PathBuf>,
+    /// output file name
+    #[structopt(
+        short = "o",
+        long = "output",
+        default_value = "cosmogony.json",
```

Comment:
This default value causes panic, as `.json` format is not supported as the output format. It should be removed or replaced with a supported extension.